configure.ac

```diff
@@ -6,7 +6,7 @@
 # Use and distribution licensed under the BSD license.  See
 # the COPYING file in this directory for full text.
 
-AC_INIT([libdrizzle],[5.1.4],[http://bugs.launchpad.net/drizzle],[libdrizzle],[http://drizzle.org/])
```

Comment:
Well if its possible to get it from the git tag that would be great but if not we can make changing it part of the release process. The problem is it would be very easy to forget.